website/notifications/emails.py

```diff
@@ -78,7 +80,17 @@ def email_digest(subscribed_user_ids, uid, event, **context):
 }
 
 
-def notify(uid, event, **context):
+def notify(uid, event, user, node, timestamp, **context):
+    """
+    :param uid: node's id
+    :param event: type of notification
+    :param user: user triggering notification
+    :param node: the node
+    :param timestamp: time
+    :param context: optional variables specific to templates
+        target_user: used with comment_replies
```

Comment:
@sloria, it is used to set:
```event = 'comment_replies' if context.get('target_user') == recipient else event```

Which is in the python, but optional since if it isn't there it is handled.

Otherwise the templates should document the needed variables.